### PR TITLE
Upload pvc-autoresizer

### PR DESCRIPTION
**Pull request by f10atin9**
Signed-off-by: f10atin9 <f10atin9@kubesphere.io>

### CODE REVIEW

File: src/main/pvc-autoresizer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+image:
+  # image.repository -- pvc-autoresizer image repository to use.
+  repository: f10atin9/pvc-autoresizer
```
Comment (stoneshi-yunify) [inline src/main/pvc-autoresizer/values.yaml:3]: this repo is wrong

File: src/main/pvc-autoresizer/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: pvc-autoresizer
+description: Auto-resize PersistentVolumeClaim objects based on Prometheus metrics.
+home: https://github.com/topolvm/pvc-autoresizer
+sources:
+  - https://github.com/topolvm/pvc-autoresizer
```
Comment (stoneshi-yunify) [inline src/main/pvc-autoresizer/Chart.yaml:6]: should use kubesphere instead of topolvm

File: src/main/pvc-autoresizer/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.3.2
```
Comment (stoneshi-yunify) [inline src/main/pvc-autoresizer/Chart.yaml:21]: should start from 0.1.0

**Review by stoneshi-yunify — CHANGES_REQUESTED** [review]

**Review by stoneshi-yunify — COMMENTED** [review]
/lgtm

**Comment by zheng1** [comment]
@xyz-li PTAL at this https://github.com/kubesphere/helm-charts/pull/205/checks

**Pull request merged**